File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/roomReward/RoomRewardModel.java
package com.syahputrareno975.cardbattlemodule.model.roomReward;

import com.syahputrareno975.cardbattlemodule.model.card.CardModel;
import java.io.Serializable;
import java.util.ArrayList;

public class RoomRewardModel  implements Serializable {
    public ArrayList<CardModel> CardsReward = new ArrayList<CardModel>();
    public Long CashReward = 0L;
    public Long ExpReward  = 0L;

    public RoomRewardModel() {
    }

    public RoomRewardModel(ArrayList<CardModel> cardsReward, Long cashReward, Long expReward) {
        CardsReward = cardsReward;
        CashReward = cashReward;
        ExpReward = expReward;
    }
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/roomReward/RoomRewardModel.kt
package com.syahputrareno975.simplecardbattle.model.roomReward

import com.syahputrareno975.simplecardbattle.model.card.CardModel
import java.io.Serializable

class RoomRewardModel  : Serializable {
    var CardsReward = ArrayList<CardModel>()
    var CashReward : Long = 0
    var ExpReward : Long = 0

    constructor()

    constructor(CardsReward: ArrayList<CardModel>, CashReward: Long, ExpReward: Long) {
        this.CardsReward = CardsReward
        this.CashReward = CashReward
        this.ExpReward = ExpReward
    }


}